crawl-ref/source/dat/database/randname.txt

```diff
@@ -1082,7 +1132,7 @@ heavily jewelled
 
 ichor-stained
 
-distressingly furry
+surprisingly fuzzy
```

Comment:
```suggestion
distressingly furry
```

Reply:
thanks for the review! when i've introducing new players to crawl, two or three different people have said something along the lines of "what's this? random subtle furry hate in this game?"